vitrina/datasets/urls.py

```diff
@@ -2,13 +2,15 @@
 
 from vitrina.datasets.views import DatasetListView
 from vitrina.datasets.views import DatasetDetailView
+from vitrina.datasets.views import DatasetSearchResultsView
 
 
 urlpatterns = [
     # @GetMapping("/datasets")
     path('datasets/', DatasetListView.as_view(), name='dataset-list'),
     # @GetMapping("/dataset/{slug}")
     path('dataset/<slug:slug>/', DatasetDetailView.as_view(), name='dataset-detail'),
+    path('datasets/search/', DatasetSearchResultsView.as_view(), name='dataset-search-results')
```

Comment:
No need for separate view, search should be implemented on top of `datasets/`, like here: https://data.gov.lt/datasets?q=duomenys